truss-chains/truss_chains/definitions.py

```diff
@@ -412,7 +413,7 @@ class DeploymentContext(SafeModelNonSerializable, Generic[UserConfigT]):
 
     data_dir: Optional[pathlib.Path] = None
     user_config: UserConfigT
-    chainlet_to_service: Mapping[str, ServiceDescriptor]
+    chainlet_to_service: dict[str, ServiceDescriptor]
```

Comment:
changed the typing from `dict` to `Mapping` here since we will now be mutating `chainlet_to_service`